sklearn/preprocessing/label.py

```diff
@@ -304,6 +304,26 @@ def fit(self, y):
         self.classes_ = unique_labels(y)
         return self
 
+    def fit_transform(self, y):
+        """Fit label binarizer and transform multi-class labels to binary labels.
+
+        The output of transform is sometimes referred to by some authors as the
+        1-of-K coding scheme.
+
+        Parameters
+        ----------
+        y : numpy array or sparse matrix of shape (n_samples,) or
+            (n_samples, n_classes) Target values. The 2-d matrix should only
```

Comment:
I think I might be missing what you mean... the indent level looks the same as in the other function definitions.

I'll drop numpy though.